music_metadata_extractor/models.py

```diff
@@ -102,14 +103,17 @@ def __repr__(self):
 class DictInput(BaseProviderInput):
     """Provider input class with a dict that contains song and artist name"""
 
-    def __init__(self, song_name: str, artist_name: str):
+    def __init__(self, song_name: str, artist_name: str, song_ytviews: int, song_ytdate: dt):
```

Comment:
I learnt something thanks to your PR! I was worried about using `int` for storing likes, but turns out Python 3 essentially renamed `long` to `int` and there is no longer a limit to the value of integers 😄 
Reference: https://docs.python.org/3.1/whatsnew/3.0.html#integers